src/bindings/python/wheel/setup.py

```diff
@@ -272,14 +274,20 @@ def generate_package(self, src_dirs):
         blacklist = [".lib", ".pdb", "_debug.dll", "_debug.dylib"]
         package_dir = os.path.join(get_package_dir(PY_INSTALL_CFG), WHEEL_LIBS_INSTALL_DIR)
 
+        exclude_patterns = ("*so.*.*.*", "*.so.*.*",
+                            "libpugixml.so", "libtbbmalloc.so", "libtbb.so",
+                            "libopenvino_*.cpython-38-x86_64-linux-gnu.so")
```

Comment:
how does it work for other python versions and architectures?

Reply:
Good point. Thanks for suggestion to use sysconfig - hardcoded extension updated, please see the current version.